src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/PasswordBox.cs

```diff
@@ -23,7 +23,7 @@ public  string PasswordChar
 		#endif
 		// Skipping already declared property Password
 		// Skipping already declared property MaxLength
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || false
```

Comment:
I think this should be false for all platforms? It seems like you've implemented it for all of them